contentreview-impl/tool/src/java/org/sakaiproject/contentreview/servlet/SubmissionCallbackServlet.java

```diff
@@ -0,0 +1,113 @@
+package org.sakaiproject.contentreview.servlet;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.JSONObject;
+
+import org.sakaiproject.assignment.api.AssignmentSubmissionEdit;
+import org.sakaiproject.assignment.cover.AssignmentService;
+import org.sakaiproject.component.cover.ComponentManager;
+import org.sakaiproject.content.cover.ContentHostingService;
+import org.sakaiproject.content.api.ContentResourceEdit;
+import org.sakaiproject.contentreview.model.ContentReviewItem;
+import org.sakaiproject.contentreview.service.ContentReviewService;
+import org.sakaiproject.entity.api.ResourcePropertiesEdit;
+import org.sakaiproject.event.cover.NotificationService;
+import org.sakaiproject.tool.api.Session;
+import org.sakaiproject.tool.cover.SessionManager;
+
+/** 
+ * This servlet will receive callbacks from TII. Then it will process the data
+ * related to the submissions and resubmission and store it.
+ */
+
+@SuppressWarnings("deprecation")
+public class SubmissionCallbackServlet extends HttpServlet {
+	
+	private static Log M_log = LogFactory.getLog(SubmissionCallbackServlet.class);
+	
+	private ContentReviewService contentReviewService =
+				(ContentReviewService)ComponentManager.get("org.sakaiproject.contentreview.service.ContentReviewService");
```

Comment:
Ah yeah, sorry that's in 11, not 10.

On 25 February 2016 at 11:00, bgarciaentornos <notifications@github.com>
wrote:

> In
> contentreview-impl/tool/src/java/org/sakaiproject/contentreview/servlet/SubmissionCallbackServlet.java
> <https://github.com/ox-it/wl-turnitin/pull/12#discussion_r54079175>:
>
> > +import org.sakaiproject.event.cover.NotificationService;
> > +import org.sakaiproject.tool.api.Session;
> > +import org.sakaiproject.tool.cover.SessionManager;
> > +
> > +/**
> > + * This servlet will receive callbacks from TII. Then it will process the data
> > + * related to the submissions and resubmission and store it.
> > + */
> > +
> > +@SuppressWarnings("deprecation")
> > +public class SubmissionCallbackServlet extends HttpServlet {
> > +	
> > +	private static Log M_log = LogFactory.getLog(SubmissionCallbackServlet.class);
> > +	
> > +	private ContentReviewService contentReviewService =
> > +				(ContentReviewService)ComponentManager.get("org.sakaiproject.contentreview.service.ContentReviewService");
>
> Changed it all, but it's not compiling without the cast.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ox-it/wl-turnitin/pull/12/files#r54079175>.
>



-- 
  Matthew Buckett, VLE Developer, IT Services, University of Oxford